From the `ansible-test sanity` stuff `make test_collection_sanity` I was seeing these

```
plugins/modules/tower_credential_input_source.py:75:1: E302: expected 2 blank lines, found 1
test/awx/test_credential_input_source.py:8:1: E302: expected 2 blank lines, found 1
test/awx/test_credential_input_source.py:10:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:22:21: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:25:17: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:33:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:67:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:73:21: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:77:17: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:84:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:118:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:124:21: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:128:17: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:135:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:172:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:178:21: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:182:17: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:189:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:227:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:233:21: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:238:17: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:245:7: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:282:21: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:285:17: E126: continuation line over-indented for hanging indent
test/awx/test_credential_input_source.py:288:1: E302: expected 2 blank lines, found 1
test/awx/test_credential_input_source.py:290:14: E225: missing whitespace around operator
test/awx/test_credential_input_source.py:292:7: E225: missing whitespace around operator
```

I can help you reproduce this if you need.

--- a/awx_collection/plugins/modules/tower_credential_input_source.py
+++ b/awx_collection/plugins/modules/tower_credential_input_source.py
@@ -72,6 +72,7 @@ EXAMPLES = '''
 
 from ..module_utils.tower_api import TowerModule
 
+
 def main():
     # Any additional arguments that are not fields of the item can be added here
     argument_spec = dict(

--- a/awx_collection/plugins/modules/tower_credential_input_source.py
+++ b/awx_collection/plugins/modules/tower_credential_input_source.py
@@ -36,7 +36,7 @@ options:
       description:
         - A JSON or YAML string
       required: False
-      type: str
+      type: dict
     target_credential:
       description:
         - The credential which will have its input defined by this source
@@ -80,7 +80,7 @@ def main():
         input_field_name=dict(required=True),
         target_credential=dict(required=True),
         source_credential=dict(default=''),
-        metadata=dict(type=dict),
+        metadata=dict(type="dict"),
         state=dict(choices=['present', 'absent'], default='present'),
     )
 

--- a/awx_collection/test/awx/test_credential_input_source.py
+++ b/awx_collection/test/awx/test_credential_input_source.py
@@ -5,9 +5,10 @@ import pytest
 
 from awx.main.models import CredentialInputSource, Credential, CredentialType, Organization
 
+
 @pytest.fixture
 def aim_cred_type():
-    ct=CredentialType.defaults['aim']()
+    ct = CredentialType.defaults['aim']()
     ct.save()
     return ct
 
@@ -19,16 +20,16 @@ def source_cred_aim(aim_cred_type):
         name='CyberArk AIM Cred',
         credential_type=aim_cred_type,
         inputs={
-                    "url": "https://cyberark.example.com",
-                    "app_id": "myAppID",
-                    "verify": "false"
-                }
+            "url": "https://cyberark.example.com",
+            "app_id": "myAppID",
+            "verify": "false"
+        }
     )
 
 
 @pytest.mark.django_db
 def test_aim_credential_source(run_module, admin_user, organization, source_cred_aim, silence_deprecation):
-    ct=CredentialType.defaults['ssh']()
+    ct = CredentialType.defaults['ssh']()
     ct.save()
     tgt_cred = Credential.objects.create(
         name='Test Machine Credential',
@@ -62,23 +63,23 @@ def test_aim_credential_source(run_module, admin_user, organization, source_cred
 @pytest.fixture
 def source_cred_conjur(organization):
     # Make a credential type which will be used by the credential
-    ct=CredentialType.defaults['conjur']()
+    ct = CredentialType.defaults['conjur']()
     ct.save()
     return Credential.objects.create(
         name='CyberArk CONJUR Cred',
         credential_type=ct,
         inputs={
-                    "url": "https://cyberark.example.com",
-                    "api_key": "myApiKey",
-                    "account": "account",
-                    "username": "username"
-                }
+            "url": "https://cyberark.example.com",
+            "api_key": "myApiKey",
+            "account": "account",
+            "username": "username"
+        }
     )
 
 
 @pytest.mark.django_db
 def test_conjur_credential_source(run_module, admin_user, organization, source_cred_conjur, silence_deprecation):
-    ct=CredentialType.defaults['ssh']()
+    ct = CredentialType.defaults['ssh']()
     ct.save()
     tgt_cred = Credential.objects.create(
         name='Test Machine Credential',
@@ -112,23 +113,23 @@ def test_conjur_credential_source(run_module, admin_user, organization, source_c
 @pytest.fixture
 def source_cred_hashi_secret(organization):
     # Make a credential type which will be used by the credential
-    ct=CredentialType.defaults['hashivault_kv']()
+    ct = CredentialType.defaults['hashivault_kv']()
     ct.save()
     return Credential.objects.create(
         name='HashiCorp secret Cred',
         credential_type=ct,
         inputs={
-                    "url": "https://secret.hash.example.com",
-                    "token": "myApiKey",
-                    "role_id": "role",
-                    "secret_id": "secret"
-                }
+            "url": "https://secret.hash.example.com",
+            "token": "myApiKey",
+            "role_id": "role",
+            "secret_id": "secret"
+        }
     )
 
 
 @pytest.mark.django_db
 def test_hashi_secret_credential_source(run_module, admin_user, organization, source_cred_hashi_secret, silence_deprecation):
-    ct=CredentialType.defaults['ssh']()
+    ct = CredentialType.defaults['ssh']()
     ct.save()
     tgt_cred = Credential.objects.create(
         name='Test Machine Credential',
@@ -165,23 +166,23 @@ def test_hashi_secret_credential_source(run_module, admin_user, organization, so
 @pytest.fixture
 def source_cred_hashi_ssh(organization):
     # Make a credential type which will be used by the credential
-    ct=CredentialType.defaults['hashivault_ssh']()
+    ct = CredentialType.defaults['hashivault_ssh']()
     ct.save()
     return Credential.objects.create(
         name='HashiCorp ssh Cred',
         credential_type=ct,
         inputs={
-                    "url": "https://ssh.hash.example.com",
-                    "token": "myApiKey",
-                    "role_id": "role",
-                    "secret_id": "secret"
-                }
+            "url": "https://ssh.hash.example.com",
+            "token": "myApiKey",
+            "role_id": "role",
+            "secret_id": "secret"
+        }
     )
 
 
 @pytest.mark.django_db
 def test_hashi_ssh_credential_source(run_module, admin_user, organization, source_cred_hashi_ssh, silence_deprecation):
-    ct=CredentialType.defaults['ssh']()
+    ct = CredentialType.defaults['ssh']()
     ct.save()
     tgt_cred = Credential.objects.create(
         name='Test Machine Credential',
@@ -219,24 +220,24 @@ def test_hashi_ssh_credential_source(run_module, admin_user, organization, sourc
 @pytest.fixture
 def source_cred_azure_kv(organization):
     # Make a credential type which will be used by the credential
-    ct=CredentialType.defaults['azure_kv']()
+    ct = CredentialType.defaults['azure_kv']()
     ct.save()
     return Credential.objects.create(
         name='Azure KV Cred',
         credential_type=ct,
         inputs={
-                    "url": "https://key.azure.example.com",
-                    "client": "client",
-                    "secret": "secret",
-                    "tenant": "tenant",
-                    "cloud_name": "the_cloud",
-                }
+            "url": "https://key.azure.example.com",
+            "client": "client",
+            "secret": "secret",
+            "tenant": "tenant",
+            "cloud_name": "the_cloud",
+        }
     )
 
 
 @pytest.mark.django_db
 def test_azure_kv_credential_source(run_module, admin_user, organization, source_cred_azure_kv, silence_deprecation):
-    ct=CredentialType.defaults['ssh']()
+    ct = CredentialType.defaults['ssh']()
     ct.save()
     tgt_cred = Credential.objects.create(
         name='Test Machine Credential',
@@ -273,15 +274,16 @@ def source_cred_aim_alt(aim_cred_type):
         name='Alternate CyberArk AIM Cred',
         credential_type=aim_cred_type,
         inputs={
-                    "url": "https://cyberark-alt.example.com",
-                    "app_id": "myAltID",
-                    "verify": "false"
-                }
+            "url": "https://cyberark-alt.example.com",
+            "app_id": "myAltID",
+            "verify": "false"
+        }
     )
+
 
 @pytest.mark.django_db
 def test_aim_credential_source(run_module, admin_user, organization, source_cred_aim, source_cred_aim_alt, silence_deprecation):
-    ct=CredentialType.defaults['ssh']()
+    ct = CredentialType.defaults['ssh']()
     ct.save()
     tgt_cred = Credential.objects.create(
         name='Test Machine Credential',

